File: FirstGameInJavaKotlin[2022]/zad2/Lab1_b_Java/app/src/main/java/com/example/lab1java/MainActivity.java
package com.example.lab1java;

import androidx.appcompat.app.AppCompatActivity;

import android.os.Bundle;
import android.view.View;
import android.widget.TextView;
import android.widget.Toast;

import org.w3c.dom.Text;

import java.util.Random;

public class MainActivity extends AppCompatActivity {
    private int r1;
    private int r2;
    private int score;
    Random rand = new Random();

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
        score = 0;
        roll();
    }

    public void roll(){
        TextView punkty = findViewById(R.id.points);
        TextView Button = findViewById(R.id.Click);

        punkty.setText("Punkty: " + score);
        r1 = rand.nextInt(10);
        r2 = rand.nextInt(10);
        Button.setText("" + r1);
    }

    public void buttonLeft(View view) {
        if(r1>=r2){
            score++;
            Toast.makeText(this, "Dobrze!!!", Toast.LENGTH_SHORT).show();
        }
        else {
            score--;
            Toast.makeText(this, "Źle!!!", Toast.LENGTH_SHORT).show();
        }
        roll();
    }

    public void buttonRight(View view) {
        if(r1<=r2){
            score++;
            Toast.makeText(this, "Dobrze!!!", Toast.LENGTH_SHORT).show();
        }
        else {
            score--;
            Toast.makeText(this, "Źle!!!", Toast.LENGTH_SHORT).show();
        }
        roll();
    }
}

File: FirstGameInJavaKotlin[2022]/zad1/Lab1_a_Kotlin/app/src/main/java/com/example/lab1kotlin/MainActivity.kt
package com.example.lab1kotlin

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.view.View
import android.widget.TextView
import android.widget.Toast
import kotlin.random.Random

class MainActivity : AppCompatActivity() {

    private var r1 = 0
    private var r2 = 0
    private var score = 0


    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
        score = 0
        roll()
    }

    fun roll(){
        findViewById<TextView>(R.id.points).text = "Punkty: " + score
        r1 = Random.nextInt(10)
        r2 = Random.nextInt(10)
        findViewById<TextView>(R.id.leftClick).text = "" + r1
        findViewById<TextView>(R.id.rightClick).text = "" + r2
    }

    fun buttonLeft(view: View) {
        if (r1>=r2){
            score++
            Toast.makeText(this, "Dobrze!!!", Toast.LENGTH_SHORT).show()
        }
        else{
            score--
            Toast.makeText(this, "Źle!!!", Toast.LENGTH_SHORT).show()
        }
        roll()
    }
    fun buttonRight(view: View) {
        if (r1<=r2){
            score++
            Toast.makeText(this, "Dobrze!!!", Toast.LENGTH_SHORT).show()
        }
        else{
            score--
            Toast.makeText(this, "Źle!!!", Toast.LENGTH_SHORT).show()
        }
        roll()
    }
}